packages/fxa-settings/src/pages/ResetPassword/en.ftl

```diff
@@ -0,0 +1,14 @@
+## ResetPassword page
+
+# If more appropriate in a locale, this can stand alone as "Continue to account settings"
+reset-password-heading-w-default-service = Reset password <span>to continue to account settings</span>
+# If more appropriate in a locale, this can stand alone as "Continue to { $serviceName }"
+# { $serviceName } represents a product name (e.g., Mozilla VPN) that will be passed in as a variable
+reset-password-heading-w-custom-service = Reset password{" "}<span>to continue to { $serviceName }</span>
+
+reset-password-warning-message = <span>Note:{" "}</span>When you reset your password, you reset your account. You may lose some of your personal information (including history, bookmarks, and passwords). That’s because we encrypt your data with your password to protect your privacy. You’ll still keep any subscriptions you may have and { product-pocket } data will not be affected.
```

Comment:
`{" "}`
Copying @flodolo 's comments form the other comment thread:
We should always use unicode characters without encoding them in FTL files. That's going to create unnecessary confusion for localizer. Out of curiosity, why inside the span and not outside?

Reply:
@vpomerleau If this was needed to make the new l10n tests pass, please add a note to that l10n-related follow up ticket you made and we can skip the test here for now and it can be re-added over there.

Reply:
@bcolsson To confirm, `{" "}` is correct?
I don't have strong reasoning for inside/outside the span. If outside is best, I'm happy to move it out.

Reply:
```suggestion
reset-password-heading-w-custom-service = Reset password{" "}<span>to continue to { $serviceName }</span>

reset-password-warning-message = <span>Note: </span>When you reset your password, you reset your account. You may lose some of your personal information (including history, bookmarks, and passwords). That’s because we encrypt your data with your password to protect your privacy. You’ll still keep any subscriptions you may have and { product-pocket } data will not be affected.
```

There is no need for the `{""}` part, you can put directly the utf8 character in the string.

Reply:
That syntax is only needed for special characters (a space, unless it's at the beginning or end of the line, is not one)
https://projectfluent.org/fluent/guide/special.html

Reply:
@flodolo : not to make the conversation go around in circles, but in the URL linked:

> Quoted text should be used sparingly, most often in scenarios which call for a special character, or when enclosing characters in {" and "} makes them easier to spot. For instance, the non-breaking space character looks like a regular space in most text editors and it's easy to miss in a translation. A Unicode escape sequence inside of a quoted text may be used to make it stand out:
> 
> privacy-label = Privacy{"\u00A0"}Policy
>

Reply:
Good point, I forgot that was there. The reality is that nowadays both text editors and Pontoon clearly differentiate non-breaking spaces from standard spaces.

I don't think we want to necessarily force all locales to use this character, so there is no point in making sure they see it. Also, pretty sure that encoding will get lost at some point anyway.